explore/src/main/scala/explore/config/SUITable.scala

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.config
+
+import cats.syntax.all._
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import react.semanticui.collections.table._
+import reactST.reactTable._
+import reactST.reactTable.mod.{ ^ => _, _ }
+import reactST.reactTable.util
+import reactST.reactTable.syntax._
+
+import scalajs.js
+import scalajs.js.|
+
+case class SUITableProps[D, TableOptsD <: UseTableOptions[D]](
```

Comment:
This seems generally useful,'will we later move it to lucuma-ui?

Reply:
I was thinking of making it a module in `scalajs-react-table`.